spec/models/resource_spec.rb

```diff
@@ -18,4 +18,39 @@
   end
 
   it_behaves_like 'indexable', :resource
+
+  describe 'privacy' do
+    let(:resource) { create(:resource, privacy: :priv) }
+
+    describe 'after_commit on: [:create]' do
+      it "adds the #{name} to the search index after creation" do
+        allow(AddToIndexJob).to receive(:perform_async)
```

Comment:
Strange -- even though the expectation is to *not* receive the stub. Oh well, works for me.